zap/src/main/java/org/zaproxy/zap/spider/parser/SpiderHtmlParser.java

```diff
@@ -217,25 +224,78 @@ private boolean parseSource(HttpMessage message, Source source, int depth, Strin
             resourcesfound |= processAttributeElement(message, depth, baseURL, el, "dynsrc");
         }
 
+        // Process IMPORT elements
+        elements = source.getAllElements(IMPORT_TAG);
+        for (Element el : elements) {
+            resourcesfound |=
+                    processAttributeElement(message, depth, baseURL, el, "implementation");
+        }
+
+        // Process content of container tags which hold text
+        List<String> elementsWithText =
+                Arrays.asList(
+                        HTMLElementName.P,
+                        HTMLElementName.TITLE,
+                        HTMLElementName.H1,
+                        HTMLElementName.H2,
+                        HTMLElementName.H3,
+                        HTMLElementName.H4,
+                        HTMLElementName.H5,
+                        HTMLElementName.H6,
+                        HTMLElementName.LI,
+                        HTMLElementName.BLOCKQUOTE);
```

Comment:
Should be extracted a constant, although I'm not quite sure about the choices. Are they based on real examples?

Reply:
Made it a constant. Some of them are based on real examples, like <p>, <title> and <li>. But the heading tags and blockquote ones are peculiar in my opinion too. Shall I remove them?

Reply:
Can be kept as is, we'll see how it goes.